openbb_platform/providers/federal_reserve/openbb_federal_reserve/models/treasury_rates.py

```diff
@@ -0,0 +1,99 @@
+"""FederalReserve Treasury Rates Model."""
+
+from datetime import datetime, timedelta
+from io import BytesIO
+from typing import Any, Dict, List, Optional
+
+import pandas as pd
+from numpy import nan
+from openbb_core.provider.abstract.fetcher import Fetcher
+from openbb_core.provider.standard_models.treasury_rates import (
+    TreasuryRatesData,
```

Comment:
Question: This data is effectively `us_yield_curve` with a start/end date.  Should these two functions be merged?

Reply:
I asked the team this exact thing and there wasnt a clear consensus.  I just went to treasury rates because it was basically the same data but from the source.

I am open to merging the two at a future point.